templates/release-docs-changelog.yml

```diff
@@ -0,0 +1,78 @@
+spec:
+  inputs:
+    stage:
+      description: The pipeline stage where to perform the docs release
+      default: publish
+    runner:
+      description: GitLab CI runner tag
+      default: hetzner-amd-beefy
+    alpine-git-tag:
+      description: Tag for alpine/git container image
+      default: latest
+    remote_changelog_file:
+      description: Remote changelog file to be used
+---
+
+
+release:mender-docs-changelog:
+  stage: $[[ inputs.stage ]]
+  tags:
+    - $[[ inputs.runner ]]
+  needs: []
+  image:
+    name: alpine/git:$[[ inputs.alpine-git-tag ]]
+    entrypoint: [""]
+  rules:
+    # Only make available for stable tags
+    - if: '$CI_COMMIT_TAG =~ /^v?\d+\.\d+\.\d+$/'
+      allow_failure: true
+  variables:
+    GITHUB_CHANGELOG_REPO_URL: "mender-docs-changelog"
+  before_script:
+    # Check required input parameters
+    - |
+      if [ -z "$[[ inputs.remote_changelog_file ]]" ]; then
+        echo "ERROR: Missing template argument 'remote_changelog_file'"
+        echo "Please specify the remote changelog name in your include section:"
+        echo "  inputs:"
+        echo "    remote_changelog_file: PATH_TO_CHANGELOG_FILE"
+        exit 1
+      fi
+    # Check required environment variables
+    - |
+      if [ -z "$GITHUB_USER_EMAIL" ] || [ -z "$GITHUB_USER_NAME" ] || [ -z "$GITHUB_CLI_TOKEN" ] || [ -z "$GITHUB_BOT_TOKEN_REPO_FULL" ]; then
+        echo "ERROR: Missing required environment variables:"
+        echo "  - GITHUB_USER_EMAIL: ${GITHUB_USER_EMAIL:-'NOT SET'}"
+        echo "  - GITHUB_USER_NAME: ${GITHUB_USER_NAME:-'NOT SET'}"
```

Comment:
This is also my bad. I am sorry. These variables are actually not set by the Mender group in CI but they were set by the [original pipeline code](https://github.com/mendersoftware/mender-gateway/blob/a002f2c151a8340d7dc46881869feeed7487ce89/.gitlab/release.yaml#L9).

I made the mistake of introducing it and we propagated it everywhere. I fixed it for my CI/CD components here: https://github.com/mendersoftware/mendertesting/pull/389/commits/c0f2bca30267ab1e9f6c0d731754a811a02a6ba4